### ISSUE TITLE
Elements on the sides of the song select screen are cut off in 5:4 aspect ratio

### ISSUE THREAD

**Issue by zetaPRIME** [dguzek/Simply-Love-SM5#52]
The banner, "artist" label and such go offscreen slightly when using 5:4 aspect ratio (my DIY "ghettocab" currently uses an old 1280x1024 monitor; it's not the biggest deal ever but a fix would be greatly appreciated!)

**Comment by dguzek** [comment]
I'll admit, I had never heard of a 5:4 aspect ratio monitor before!

This will be fairly low on my priority list, but I'll see what I can do.

**Comment by zetaPRIME** [comment]
Yeah, 1280x1024 monitors were popular for a relatively short time before widescreen became the norm.

**Comment by TheKamilia** [comment]
ill check it out, i can probably emulate a 5:4 aspect ratio on my computer
